docker-compose.override.yml

```diff
@@ -0,0 +1,6 @@
+version: '3.4'
+
+services:
+  app:
+    build:
+      target: dev
```

Comment:
Might be better to call the file `docker-compose.dev.yml` as it's abstract behind Make?

Reply:
Eventually people will use custom `docker-compose` commands for debugging e.g. `docker-compose ps | grep ...` or `docker-compose logs ...`. As it stands this probably doesn't strictly need to be picked up there, but in general it should in case anything is added?

Reply:
Changes in 49c14b8a7f9f0e4a760e8a71cf8c3ea1278a7696. Docker Compose commands should go through Make, if it's just passing it through (So `make docker-compose "ps"` or something).